src/components/galleryCard/GalleryCard.types.ts

```diff
@@ -0,0 +1,16 @@
+import { LayoutProps, SpaceProps } from 'styled-system'
+
+export interface GalleryCardProps extends LayoutProps, SpaceProps {
+  artWorkTitle: string
+  artistName: string
+  date: string
+  priceOrSold: string
+  image: {
+    src: string
+    srcSet: string
+    width: number
+    height: number
+  }
+  altText: string
+  tagContent: string
+}
```

Comment:
Now the content model has been finalised, we can update the props to be the same shape so they can just be passed straight through from the model:

```ts
interface GalleryCardProps ... {
  artist: {
    name: string
  }
  artwork: {
    title: string
    year: string
    sold: boolean
    guidePrice: string
    image: FixedObject // this comes from gatsby-image
    category: Catgeory // this comes from the FilterContainer.types.ts
  }
}
```